cli/dev/service.go

```diff
@@ -27,6 +32,48 @@ func ServiceNew(homeDirectory string, org string, dconfig string) {
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, "'%v %v' must specify either --org or set the %v environment variable.", SERVICE_COMMAND, SERVICE_CREATION_COMMAND, DEVTOOL_HZN_ORG)
 	}
 
+	// check if the input version is a valid version string
```

Comment:
I would like to see input variable verification done in a separate function, similar to the way its done in dependency fetch because it keeps the mainline cleaner and the verification can be tested separately.

Reply:
done